src/services/ParticipantProjectService.ts

```diff
@@ -56,7 +56,6 @@ const getSearchParams = (
       'country_name',
       'country_region',
       'acceleratorDetails_confirmedReforestableLand',
-      ...(isCsv ? [] : ['acceleratorDetails_confirmedReforestableLand(raw)']),
```

Comment:
remove the isCsv option from function signature as well?

Reply:
Nice catch, you got it